kratos/containers/variables_list_data_value_container.h

```diff
@@ -120,14 +118,11 @@ class KRATOS_API(KRATOS_CORE) VariablesListDataValueContainer
         // Setting the current position with relative source container offset
         mpCurrentPosition = mpData + (rOther.mpCurrentPosition - rOther.mpData);
 
-        SizeType size = mpVariablesList->DataSize();
-        for(VariablesList::const_iterator i_variable = mpVariablesList->begin() ;
```

Comment:
Same comment.

Reply:
In this PR I am just moving to one line, not actually changing code